core/src/por/reporter.ts

```diff
@@ -27,10 +27,11 @@ async function shouldReport({
 
   // Check Submission Hearbeat
   const updatedAt = Number(latestRoundData.updatedAt) * 1000 // convert to milliseconds
-  const timestamp = Date.now()
+  const now = Date.now()
+  const latancyBuffer = 60000 // submission latancy buffer 1 min
   const heartbeat = aggregator.heartbeat
 
-  if (updatedAt + heartbeat < timestamp) {
+  if (updatedAt + heartbeat - latancyBuffer < now) {
```

Comment:
If `latencyBuffer` is larger than `heartbeat` bad things will start happening. We should define the `latencyBuffer` somewhere outside and make a check whether `heartbeat` is larger then `latencyBuffer` at the beginning of execution. If `heartbeat` is smaller we should no proceed further, just raise an error that will be sent to slack.

Reply:
@martinkersner Really good point. I can move the `lateanceBuffer` to core `settings.ts` file and add the comparison check with `heatbeat`.